docs/memory-bank/systemPatterns.md

```
@@ -0,0 +1,41 @@
+# System Patterns — Cognitive Dissonance
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)